lib/mailer.js

```diff
@@ -89,6 +97,15 @@ module.exports = function (config, log) {
           }
         ))
       }
+      mailer.sendSuspiciousLocationsEmail = function (email, locations, opts) {
+        return P.resolve(mailer.postVerifyEmail(
```

Comment:
This should be `mailer.suspiciousLocationEmail` rather than `postVerifyEmail`; also inconsistency between "Locations" in name here and "Location" in method name in the auth-mailer.